### PR TITLE
this fix allow the ng repeat to loop duplicated object in an array, 

### PR DESCRIPTION
**Pull request by sh977218**
just added track by $index in all the places

### CODE REVIEW

File: src/select2/match-multiple.tpl.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   do not work: [class^="select2-choice"]
 -->
 <span class="ui-select-match">
-  <li class="ui-select-match-item select2-search-choice" ng-repeat="$item in $select.selected" 
+  <li class="ui-select-match-item select2-search-choice" ng-repeat="$item in $select.selected track by $index track by $index"
```
Comment (user378230) [inline src/select2/match-multiple.tpl.html]: Duplicated `track by $index`...

**Comment by user378230** [comment]
Also, please undo changes to /dist files and squash to a single commit. Thanks :+1:

**Comment by sh977218** [comment]
@user378230  I'll fix them and do a new PR.

**Comment by sh977218** [comment]
new PR:
https://github.com/angular-ui/ui-select/pull/1688
closing this PR.

**Pull request closed without merging**